src/components/ClusterOverview/Health/Health.js

```diff
@@ -1,26 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { get } from 'lodash';
 
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
 } from '../../Dashboard/DashboardCard';
-import HealthBody from './HealthBody';
 import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
 import { InlineLoading } from '../../Loading';
+import { HEALTHY, ERROR, UNKNOWN } from './strings';
+import { HealthItem } from '../../Dashboard/Health/HealthItem';
+import { HealthBody } from '../../Dashboard/Health/HealthBody';
 
-export const Health = ({ data, loaded }) => (
-  <DashboardCard>
-    <DashboardCardHeader>
-      <DashboardCardTitle>Cluster Health</DashboardCardTitle>
-    </DashboardCardHeader>
-    <DashboardCardBody className="kubevirt-health__body" isLoading={!loaded} LoadingComponent={InlineLoading}>
-      <HealthBody data={data} />
-    </DashboardCardBody>
-  </DashboardCard>
-);
+const HealthStatus = {
```

Comment:
Please do not change data format from the previous one, we are not using the same as in OCS

Reply:
I would split the PR to two - first one would contain new generic HealthBody/Item for Dashboard + Health in StorageOverview. I will create second one which changes ClusterOverview health to use HealthBody/Item from Dashboard - as this will probably require changes in web-ui too